programmers/문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,22 @@
+function solution(s){
+    s = s.toLowerCase();
+    let p = s.match(/p/g);
+    let y = s.match(/y/g);
+    
+    if(p !=null && y != null){
+      if(p.length == y.length){
+         return true;
+      }
+      else if(p.length != y.length){
+        return false;
+        
+      }
+    }
+    if(p == null && y == null){
+      return true;
+    }  
+    else 
+        return false;
+    
+    
+}
```

Comment:
if else문에서 한줄로 작성할 경우 { }가 필요없는데 이렇게 사용할 경우 그냥 한줄로 작성하시는게 좋습니다.
그리고 공백이나 줄바꿈 좀만 신경써주시면 좋을 것 같아여
```suggestion
    if (p === null && y === null) return true;
    else return false;
}
```